packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -1019,53 +1002,23 @@ contract Deploy is Deployer {
             _gameType: GameTypes.CANNON,
             _absolutePrestate: loadMipsAbsolutePrestate(),
             _faultVm: IBigStepper(mustGetAddress("Mips")),
-            _maxGameDepth: 30 // Hard code depth for legacy game to keep e2e tests fast
-         });
-    }
-
-    /// @notice Sets the implementation for the `OUTPUT_CANNON` game type in the `DisputeGameFactory`
-    function setCannonOutputBisectionGameImplementation() public onlyDevnet broadcast {
-        console.log("Setting Cannon OutputBisectionGame implementation");
-        DisputeGameFactory factory = DisputeGameFactory(mustGetAddress("DisputeGameFactoryProxy"));
-
-        _setFaultGameImplementation({
-            _factory: factory,
-            _gameType: GameTypes.OUTPUT_CANNON,
-            _absolutePrestate: loadMipsAbsolutePrestate(),
-            _faultVm: IBigStepper(mustGetAddress("Mips")),
-            _maxGameDepth: cfg.faultGameMaxDepth()
-        });
-    }
-
-    /// @notice Sets the implementation for the `OUTPUT_ALPHABET` game type in the `DisputeGameFactory`
-    function setAlphabetOutputBisectionGameImplementation() public onlyDevnet broadcast {
-        console.log("Setting Alphabet OutputBisectionGame implementation");
-        DisputeGameFactory factory = DisputeGameFactory(mustGetAddress("DisputeGameFactoryProxy"));
-
-        Claim outputAbsolutePrestate = Claim.wrap(bytes32(cfg.faultGameAbsolutePrestate()));
-        _setFaultGameImplementation({
-            _factory: factory,
-            _gameType: GameTypes.OUTPUT_ALPHABET,
-            _absolutePrestate: outputAbsolutePrestate,
-            _faultVm: IBigStepper(new AlphabetVM2(outputAbsolutePrestate)),
             _maxGameDepth: cfg.faultGameMaxDepth()
-        });
+         });
     }
 
     /// @notice Sets the implementation for the `ALPHABET` game type in the `DisputeGameFactory`
     function setAlphabetFaultGameImplementation() public onlyDevnet broadcast {
         console.log("Setting Alphabet FaultDisputeGame implementation");
         DisputeGameFactory factory = DisputeGameFactory(mustGetAddress("DisputeGameFactoryProxy"));
 
-        // Set the Alphabet FaultDisputeGame implementation in the factory.
-        Claim alphabetAbsolutePrestate = Claim.wrap(bytes32(cfg.faultGameAbsolutePrestate()));
+        Claim outputAbsolutePrestate = Claim.wrap(bytes32(cfg.faultGameAbsolutePrestate()));
         _setFaultGameImplementation({
             _factory: factory,
             _gameType: GameTypes.ALPHABET,
-            _absolutePrestate: alphabetAbsolutePrestate,
-            _faultVm: IBigStepper(new AlphabetVM(alphabetAbsolutePrestate)),
-            _maxGameDepth: 4 // The max game depth of the alphabet game is always 4.
-         });
+            _absolutePrestate: outputAbsolutePrestate,
+            _faultVm: IBigStepper(new AlphabetVM(outputAbsolutePrestate)),
+            _maxGameDepth: cfg.faultGameMaxDepth()
```

Comment:
Can we change this to use @ajsutton's output alphabet max game depth config in https://github.com/ethereum-optimism/optimism/pull/8798

```suggestion
            // The max depth for the alphabet trace is always 4. Add 1 because split depth is fully inclusive.
            _maxGameDepth: cfg. faultGameSplitDepth() + 4 + 1
```

Reply:
Sure! Good spot.